database/config.go

```diff
@@ -76,6 +79,16 @@ func NewConfig(dataDir string, metrics bool, ctx *cli.Context) (Config, error) {
 	gethCfg.EthCfg.TrieDirtyCache = 0 // MB
 	gethCfg.EthCfg.TrieTimeout = 5 * time.Minute
 
+	// To prevent DDOS TX spam, a min gas price must be defined
```

Comment:
As you already gathered the knowledge regarding the meaning behind each of this values. I think it would be a good idea to include a comment over each of them for future references and prevent next developers to waste their time again investigating the reason for each of these assigments

Reply:
sure, i will add more explanation for each variable